### DISCUSSION TITLE
can't connect my Mid 2012 macbook pro running catalina

### DISCUSSION THREAD

**Discussion by willhyde91** [apohl79/audiogridder#1412] [Help]
Hi have have successfully connected servers on my m1 macbook (main plugin host) and an old mac mini (server) and a pc (server).
I am trying to use my old Mid 2012 macbook pro running catalina as a server over lan connected to the same switch as the others but i cannot get it to be recognized on any of my machines. i have tried uninstalling and re-installing every version of the server application for mac and no change. Any help? it's very strange i have no idea what the issue is

**Comment by willhyde91** [comment]
anybody?

**Reply by kolutshan** [reply]
Can the machines see and ping each other? Make sure that basic connection works first. Do you have any firewalls on those machines? Like LittleSnitch? Or maybe it is a DNS resolve issue - so have you tried just the IP? Also make sure to set static IP adresses to the server machines and what might also help is to add the server names and ip to your hosts file.
